pkg/scheduler/scheduler_test.go

```diff
@@ -430,17 +430,17 @@ func TestSchedule(t *testing.T) {
 			},
 			wantScheduled: []string{"eng-beta/new"},
 		},
-		"cannot borrow if cohort was assigned": {
+		"cannot borrow if cohort was assigned and will result in overadmission": {
```

Comment:
```suggestion
		"cannot borrow if cohort was assigned and would result in overadmission": {
```

Reply:
I wanted to convey that there is no actual overadmission, so the overadmission is hypothetical if we let all the workloads in. Maybe it can be phrased differently, but in the current form I think it is not clear either.